07.bowling_object/lib/shot.rb

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+STRIKE = 10
+
+class Shot
+  attr_reader :mark
+
+  def initialize(mark)
+    @mark = mark
+  end
+
+  def score
+    return STRIKE if mark == 'X'
```

Comment:
```suggestion
    return 10 if strike?
```

かなー。
10は定数化するほどではないかも、と思いました。（好みかもしれませんが）

あと僕だったらここは三項演算子で1行で書くかなー。